I13 comments

marian-vignau opened the pull request:
#13 Some issues with format remain.

marian-vignau committed:
Disqus widget added and imported posts from old version.

marian-vignau committed:
Initial commit. Added disqus comments to the site, and an import script .

marian-vignau committed:
Initial commit. Added comments with disqus widget and import comments from old site, using a custom widget

marian-vignau committed:
Merge branch 'i13-comments' of https://github.com/facundobatista/blog into i13-comments

facundobatista commented on migrate-comments.py:
```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8 -*-
```
No es necesario esto en Py3

facundobatista commented on migrate-comments.py:
```diff
@@ -0,0 +1,239 @@
+# May not be efficient for huge blogs since the script keeps
+# all content in the memory during conversion.
+#
+# Released as public domain.
```
Sacá esto, la licencia es la general del proyecto.

facundobatista commented on migrate-comments.py:
```diff
@@ -0,0 +1,239 @@
+#
+# Released as public domain.
+#
+# Required library: dateutil - https://labix.org/python-dateutil
```
Dónde se usa esto?

facundobatista commented on migrate-comments.py:
```diff
@@ -0,0 +1,239 @@
+
+import pickle
+import html
+from slugify import slugify   # fades python-slugify
```
ya que este script necesita fades, hacelo ejecutable y ponele fades en el shebang

facundobatista commented on site/themes/blogtxt/templates/post.tmpl:
```diff
@@ -45,4 +45,40 @@ ${helper.twitter_card_information(post)}
+    /*
+    var disqus_config = function () {
+        // Replace PAGE_URL with your page's canonical URL variable
+        this.page.url = PAGE_URL;
```
no tendríamos que meter el valor de la url del post, acá? para evitar threads duplicados...

facundobatista reviewed (changes requested):
Algunos cambios, genial!

También te pido por favor que expliques (en la parte de conversación del PR) qué es lo que está roto a nivel CSS por los cambios de la diseñadora, así no funciono yo como un teléfono descompuesto.

Gracias!!

marian-vignau committed:
Improved migrate script description, fixed errors.
Added shebang and make it executable.
Supressed disqus javascript from post.tmpl because it's redundant, nikola add this automatically.

marian-vignau committed:
Merge branch 'master' of https://github.com/facundobatista/blog into i13-comments

marian-vignau committed:
Erased one blank line
